python/src/block/InterpolationData.python.cpp

```diff
@@ -47,29 +47,45 @@ void wrapInterpolationData( python::module& module, python::module& ) {
     &Block::isLinearLinear,
     "Flag to indicate whether or not the data is fully linear-linear"
   )
+  .def_property_readonly(
+
+    "NB",
+    [] ( const Block& self ) { return self.NB(); },
+    "The number of interpolation regions"
+  )
+  .def_property_readonly(
+
+    "number_interpolation_regions",
+    [] ( const Block& self ) { return self.numberInterpolationRegions(); },
+    "The number of interpolation regions"
+  )
   .def_property_readonly(
 
     "NBT",
-    &Block::NBT,
+    [] ( const Block& self ) -> LongRange
+       { return self.NBT(); },
```

Comment:
NBT() returns a range and not all ranges are the same (any manipulation of a range creates another unique range type). As a result, we would have to bind an enormous amount of range types. The lambda converts this range into a range type called ```any_view< T >``` so that we only need to bind this particular type.